arviz/plots/plot_utils.py

```diff
@@ -309,7 +309,7 @@ def get_plotting_function(plot_name, plot_module, backend):
 
         except (ImportError, AssertionError) as err:
             raise ImportError(
-                "'bokeh' backend needs Bokeh (1.4.0+) installed." " Please upgrade or install"
+                "'bokeh' backend needs Bokeh (1.4.0+) installed." + " Please upgrade or install"
```

Comment:
Can't this be a single string like this? Using either space or plus sign concatenation looks overly complicated

```suggestion
                "'bokeh' backend needs Bokeh (1.4.0+) installed. Please upgrade or install"
```

Reply:
@OriolAbril yeah, of course, was thinking so, but just wasn't sure if there was a specific purpose before to concatenate string in that way, so i thought better keep as is. 

Applied the suggestion. Thanks!